lib/commands/post-install.ts

```diff
@@ -0,0 +1,91 @@
+import {PostInstallCommand} from "../common/commands/post-install";
+let emailValidatorModule = require('email-validator');
```

Comment:
you can include the email-validator.d.ts file from here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/email-validator/email-validator.d.ts
Which will allow you to use 
```TypeScript
import * as emailValidator  from "email-validator";
```

But this is not a merge stopper :)